rllib/agents/ppo/appo_tf_policy.py

```diff
@@ -48,443 +48,425 @@
 logger = logging.getLogger(__name__)
 
 
-def make_appo_model(
-    policy: Policy,
-    obs_space: gym.spaces.Space,
-    action_space: gym.spaces.Space,
-    config: TrainerConfigDict,
-) -> ModelV2:
-    """Builds model and target model for APPO.
+class MakeAPPOModel:
```

Comment:
Same here. Let's actually start NOT using mixins wherever possible, but simple method overrides.
Whenever a mixin doesn't have state, we should simply override the mixin's method  in the policy sub-class (here: make_model).

Reply:
done